frontend/src/components/header/Header.jsx

```diff
@@ -34,17 +26,17 @@ const Header = () => {
     };
     
     return (
-      <div>
-        <header className='sticky top-0 z-[1] mx-auto flex w-full max-w-2xl:max-w-7xl flex-wrap items-center justify-between bg-background p-[2em] font-sans font-bold backdrop-blur-[100px] bg-slate-900 text-white'>
-                <div className='mr-10 ml-10'>
-                  <h2 className='font-bebas text-3xl'>Event finder</h2>
+      <div >
+        <header className='fixed z-[1] top-0 w-full flex flex-wrap items-center justify-between bg-background p-[2em] font-sans font-bold backdrop-blur-[100px] bg-slate-900 text-white'>
+                <div className='m-3 items-center'>
+                  <h2 className='font-bebas text-xl text-3xl'>Event finder</h2>
                 </div>
 
-                <div className='mr-1'>
+                <div className='hidden mr-1 content-center text-black lg:flex'>
                   <SearchBar />
                 </div>
 
-                <div className='flex flex-[1] items-center justify-end flex-wrap items-center'>
+                <div className='content-center'>
                   <div className="hidden justify-end lg:flex" id='component'>
                       <NavLinksLogged />
```

Comment:
See comments on your initialHeader. The two header components can be consolidated.

Reply:
To do

Reply:
Done